File: src/Projects/Android/012-ImageProcessingServerNetLib/Server/src/main/java/org/csystem/app/imageprocessing/server/configuration/thread/ThreadPoolConfig.java
package org.csystem.app.imageprocessing.server.configuration.thread;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

import java.util.concurrent.ExecutorService;
import java.util.concurrent.Executors;

@Configuration
public class ThreadPoolConfig {
    @Bean
    public ExecutorService executorService()
    {
        return Executors.newCachedThreadPool();
    }
}

File: src/Projects/Android/013-RandomTextGeneratorServer/Client/src/main/kotlin/org/csystem/app/random/text/client/configuration/thread/ThreadPoolConfig.kt
package org.csystem.app.random.text.client.configuration.thread

import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import java.util.concurrent.ExecutorService
import java.util.concurrent.Executors

@Configuration
class ThreadPoolConfig {
    @Bean
    fun threadPool() : ExecutorService = Executors.newSingleThreadExecutor()
}